frontend/services/entities/host_query_report.ts

```diff
@@ -0,0 +1,20 @@
+import sendRequest from "services";
+import endpoints from "utilities/endpoints";
+
+export interface IHQRResult {
+  columns: Record<string, any>;
```

Comment:
Can this `any` be narrowed? Or perhaps use `unknown` instead?

Reply:
Great point, looks like it'll always be `Record<string, string>`